src/i18n/locales/ru.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bandingCenter": "Центр Кольцевания Птиц при Акажемии наук Республики Беларусь",
```

Comment:
Can we have more than 1 level? As an example `registration.password` instead of `password`?

Reply:
Yep, good idea. I propose initially give screen-specific domens to all of the translations, further if there will be doublicates, they can be put on the top level, out of the domen, are you ok with that?

Reply:
I am okay with duplicates.